packages/shared/lib/core/ledger/actions/registerLedgerDeviceEventHandlers.ts

```diff
@@ -2,8 +2,9 @@ import { get } from 'svelte/store'
 import { Platform } from '@core/app/classes'
 import { addError } from '@core/error'
 import { activeAccounts, updateActiveAccount, updateActiveAccountPersistedData } from '@core/profile'
-import { deconstructBip32Path } from '@core/account'
+import { deconstructBip32Path, updateSelectedAccount } from '@core/account'
 import { ChainId, getNetwork } from '@core/network'
+import { closePopup } from '../../../../../desktop/lib/auxiliary/popup'
```

Comment:
it is not possible to have a nicer import because this file is in `shared` and `closePopup` is in `desktop`.

on my next PR where i remove this event and add a promise, this is removed as well